bottles/frontend/views/list.py

```diff
@@ -135,7 +135,9 @@ def set_path(_dialog, response):
         add_all_filters(dialog)
         dialog.set_modal(True)
         dialog.connect("response", set_path)
-        dialog.show()
+
+        if Xdp.Portal.running_under_sandbox():
+            dialog.show()
```

Comment:
i think it would make better sense to check before building the dialog just to then not show